server/src/test/java/org/opensearch/cluster/metadata/MetadataCreateIndexServiceTests.java

```diff
@@ -611,6 +611,20 @@ public void testPrepareResizeIndexSettingsCopySettings() {
         );
     }
 
+    public void testPrepareResizeIndexSettingsReplicationType() {
```

Comment:
Could we also add some yaml rest test cases to make sure the rest api behaves as expected? Thanks!

Reply:
LGTM, FIXED At https://github.com/opensearch-project/OpenSearch/pull/16076/commits/81aec5e113bd19a9228133ecd671a7a021421eba